From: Jesper Dangaard Brouer <hawk@comx.dk>
Date: Tue, 24 Jan 2012 16:03:33 -0500
Subject: [PATCH 3/3] net: flow_dissector.c missing include linux/export.h

commit c452ed70771cea3af73d21a5914989137fbd28b8 upstream.

The file net/core/flow_dissector.c seems to be missing
including linux/export.h.

Signed-off-by: Jesper Dangaard Brouer <hawk@comx.dk>
Signed-off-by: David S. Miller <davem@davemloft.net>

Index: linux-3.2.46/net/core/flow_dissector.c
===================================================================
--- linux-3.2.46.orig/net/core/flow_dissector.c	2013-07-26 19:22:28.000000000 +0000
+++ linux-3.2.46/net/core/flow_dissector.c	2013-07-26 19:22:30.000000000 +0000
@@ -1,4 +1,5 @@
 #include <linux/skbuff.h>
+#include <linux/export.h>
 #include <linux/ip.h>
 #include <linux/ipv6.h>
 #include <linux/if_vlan.h>
